packages/TelegramClient-UI.package/TCUChatsList.class/instance/chatItemClicked.event.from..st

```diff
@@ -1,13 +1,10 @@
 event handling
 chatItemClicked: aChatListItem event: anEvent from: aSender
 
-	self deselectChats.
-
 	(aChatListItem chatID ~= self selectedChat id)
```

Comment:
why not `... = ... ifTrue:` ? why does it have to be negated here?

Reply:
Yeah good question ^^
I will change it when we decide how to handle the other comment on this method.